fxmanifest.lua

```diff
@@ -32,3 +32,6 @@ provides {
     'toko-voip',
     'tokovoip_script'
 }
+
+export 'getRadioVolume'
```

Comment:
we do exports async, so exports should go into the actual file

Reply:
What difference does it make, or is it just to keep it the same way?

Reply:
better readability + able to keep everything in one spot